lib/crypto.ts

```diff
@@ -0,0 +1,32 @@
+import crypto from 'crypto'
+
+const ENCRYPTION_KEY = Buffer.from(process.env.ENCRYPTION_KEY || '', 'hex')
+const ALGORITHM = 'aes-256-cbc'
+const IV_LENGTH = 16
+
+if (!ENCRYPTION_KEY || ENCRYPTION_KEY.length !== 32) {
```

Comment:
The code throws an error if `ENCRYPTION_KEY` is not set, but `.env.example` marks it as "Optional". This will crash the app on startup if the user hasn't set this variable

Anyone deploying this branch without the `ENCRYPTION_KEY` will experience immediate crashes

Recommendation: Make it truly optional (only validate when MCP features are used)

Note: `.env.example` has been removed to prevent commits with sensitive info. Please update README instead.